### ISSUE TITLE
Fold/Unfold for tab tree?

### ISSUE THREAD

**Issue by dsvi** [traxium/tabtree#205]
Hi!
Is it possible to add fold/unfold functionality to the tree?

**Comment by hollusion** [comment]
This feature is very important!

**Comment by mjmoore** [comment]
I normally like to hide some collection of tabs to come back to another time. As Tab Groups is not recommended with this add-on, then folding/collapsing of tabs would be a great addition.

**Comment by Asmageddon** [comment]
Yeah this is my main issue with this addon. Tab trees aren't very useful without the ability to collapse them.
